extensions/interactions/base.py

```diff
@@ -186,6 +187,7 @@ def to_dict(self):
                 'schema': ca_spec.schema,
             } for ca_spec in self.customization_arg_specs],
             'instructions': self.instructions,
+            'narrow_instructions': self.narrow_instructions,
```

Comment:
Please add a test to base_test.py that ensures that, for all interactions, this field is not None.